### PR TITLE
Wait for clickhouse-server to start in stateful tests

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: docker/test/stateful/run.sh [hunk 1]
```diff
@@ -36,10 +36,26 @@ echo "ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environ
 echo "UBSAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 echo "LLVM_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 
-service zookeeper start
```
Comment (alexey-milovidov) [inline docker/test/stateful/run.sh:39]: Also removed ZooKeeper (we don't use it for functional tests from 2018).

File: docker/test/stateful/run.sh [hunk 2]
```diff
@@ -36,10 +36,26 @@ echo "ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environ
...
+    do
+        if [ "$counter" -gt 120 ]
+        then
+            echo "Cannot start clickhouse-server"
+            cat /var/log/clickhouse-server/stdout.log
+            tail -n1000 /var/log/clickhouse-server/stderr.log
+            tail -n1000 /var/log/clickhouse-server/clickhouse-server.log
+            break
+        fi
+        timeout 120 service clickhouse-server start
```
Comment (alesapin) [inline docker/test/stateful/run.sh:52]: 120 retries and 120 seconds timeout. `120 * 120` = 4 hours. Isn't it too much?

**Comment by alexey-milovidov** [comment]
> test_storage_rabbitmq/test.py::test_rabbitmq_headers_exchange

@alesapin flaky test.

File: docker/test/stateful/run.sh (hunk 2 above)
Comment (alexey-milovidov) [reply docker/test/stateful/run.sh:52]: Yes, it's too much. The same logic is used in stress test.
I don't expect inner timeout will be needed on every loop iteration.

File: docker/test/stateful/run.sh [hunk 3]
```diff
@@ -36,10 +36,26 @@ echo "ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environ
 echo "UBSAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 echo "LLVM_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
```
Comment (qoega) [inline docker/test/stateful/run.sh:37]: You can remove this also as symbolizer paths are set in test base docker image. Stateless and stateful tests use it as common ancestor 
https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/base/Dockerfile#L53-L55

Comment (alexey-milovidov) [reply docker/test/stateful/run.sh]: Ok.

File: docker/test/stateful/run.sh (hunk 2 above)
Comment (alesapin) [reply docker/test/stateful/run.sh:45]: ok

**Pull request merged**